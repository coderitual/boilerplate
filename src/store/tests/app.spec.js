import reducer, { addUser } from '../app';

test('app reducer has correct inital value', () => {
  expect(reducer(undefined, { type: 'FAKE' })).toMatchSnapshot();
});

test('app reducer adds user', () => {
  expect(reducer(undefined, addUser('mike'))).toMatchSnapshot();
});
