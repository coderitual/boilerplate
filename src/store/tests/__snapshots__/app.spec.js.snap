// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app reducer adds user 1`] = `
Array [
  "mike",
]
`;

exports[`app reducer has correct inital value 1`] = `Array []`;
